IdentityCore/src/broker_operation/request/silent_token_request/MSIDBrokerOperationSilentTokenRequest+SSORequest.h

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+#import "MSIDBrokerOperationSilentTokenRequest.h"
+#import "MSIDSSORequestProviding.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface MSIDBrokerOperationSilentTokenRequest (SSORequest) <MSIDSSORequestProviding>
```

Comment:
This can be part of the class itself, or is there a reason for it to be in the category?
If there's no reason, let's start from it being part of the class, and move out to categories if the first strategy doesn't work out (e.g. class becomes too big and difficult to follow)? I personally think these classes will be fairly small, so should be quite readable even without splitting.

Reply:
This category was removed in this [pr](https://github.com/AzureAD/microsoft-authentication-library-common-for-objc/pull/588).